basetest.pm

```diff
@@ -622,9 +622,11 @@ sub rollback_activated_consoles {
         autotest::query_isotovideo('backend_reset_console', {testapi_console => $console});
     }
 
-    my $ret = autotest::query_isotovideo('backend_select_console',
-        {testapi_console => $autotest::last_milestone_console});
-    die $ret->{error} if $ret->{error};
+    if (defined($autotest::last_milestone_console)) {
```

Comment:
This makes sense to me, but @coolo is usually good at having hunches :)
@richiejp did you manage to get a job passing with this fix? Just to be on the safe side :)

Reply:
This job was crashing after failing, but now it reverts the image successfully... then ends. It is king of dump, but it proves this change works: http://10.160.67.78/tests/561

Reply:
@richiejp but we still would need to set up a default console at a test distribution level, right?